src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/AcceptanceTypeMapper.java

```diff
@@ -0,0 +1,36 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import com.google.common.collect.ImmutableMap;
+import org.hl7.fhir.r4.model.Parameters;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.AcceptanceType;
+
+import java.util.Map;
+
+public class AcceptanceTypeMapper implements FromFhirToEdifactMapper<AcceptanceType> {
+    private final static String ACCEPTANCE_TYPE = "acceptanceType";
+    private final static Map ACC_TYPE_MAPPING = ImmutableMap.of(
+            "1", "Birth",
+            "2", "1st Acceptance",
```

Comment:
Is "1st Acceptance" that value that will be used on the FHIR API payload? OpenAPI document suggests more "machine" type values e.f. birth, first, transferin, etc...

Should these be case sensitive or case-insensitive?

Reply:
I've taken that from 'PCRM - NHAIS EDIFACT Messages Analysis' doc page 7.
Not sure if it's our input.
Currently a value between 1 and 5:
1 - Birth
2 – 1st Acceptance
3 – Transfer-in
4 - Immigrant
5 – Ex-services

Reply:
The number is the identifier that goes into the EDIFACT message, the rest is just a label/description.

I'd suggest using the text identifiers from the openapi spec.